dbt_platform_helper/utils/validation.py

```diff
@@ -328,12 +329,7 @@ def iam_role_arn_regex(key):
 S3_DEFINITION.update(
     {
         "type": "s3",
-        Optional("objects"): [
-            {
-                "key": str,
-                Optional("body"): str,
-            }
-        ],
+        Optional("objects"): [{"key": str, Optional("body"): str, Optional("content_type"): str}],
```

Comment:
Unless content type is specified, the objects default to "binary/octet-stream". This means that browsers will download an html file rather displaying in browser

Reply:
Worth mentioning this in the documentation part.